usage.md

```diff
@@ -102,6 +102,21 @@ After a file is successfully uploaded to the Arcana Store, it is assigned a uniq
 let shareURL = await dAppStorageProvider.files.getPublicFileURL(did);
 ``` -->
 
+### Upload Chunk Size
+
+Arcana Storage SDK uses chunked uploading mode. Each file is split into chunks of default size 10MB and chunks are uploaded one by one. These chunks are further processed on the server, encrypted, and split into multiple parts before being stored in the Arcana Store. Chunked uploading helps to overcome the server's default upload size limitation and there is no server-side setting change required to upload a very large file size. Also, it helps in decentralizing data storage.
```

Comment:
> Uses chunked uploading mode

What does that even mean? I suggest: “uploads files in chunks”.

>chunks of default size 10MB

I recommend: “chunks (10 MiB by default)”.